common/diagnostics/Diagnostics.java

```diff
@@ -128,16 +144,25 @@ protected static void initSentry(String serverID, String distributionName, Strin
         }
 
         @Override
-        public void mayStartServing(@Nullable SslContext sslContext, ChannelInboundHandlerAdapter... middleware) {
+        public void mayStartMonitoring(@Nullable SslContext sslContext, ChannelInboundHandlerAdapter... middleware) {
             if (monitoringServer != null) monitoringServer.startServing(sslContext, middleware);
         }
```

Comment:
The monitoring server doesn't monitor, it serves the data which the user can use to monitor the statistics.

Reply:
`mayStartMonitoringService` it is =)

with the `serve` word it is really unclear from the outside if it's about reporting or monitoring.